soccer/motion/MotionSetpoint.hpp

```diff
@@ -1,9 +1,9 @@
 #pragma once
 // stores the outputs published by MotionControl
 struct MotionSetpoint {
-    float xvelocity;
-    float yvelocity;
-    float avelocity;
+    double xvelocity;
+    double yvelocity;
+    double avelocity;
```

Comment:
is there a reason for this change? Aren't floats used everywhere though?

Reply:
I mean, yes they are in a lot of places, but that doesn't really mean they should be. There's really no reason not to use doubles. I changed some motion control code to use double, so I figured I might as well change this too.

Reply:
I guess we should make an issue to just straight up change all floats to doubles then

Reply:
Probably